app/models/user.rb

```diff
@@ -2,4 +2,12 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   enum role: %i[user manager admin]
+
+  validate :validate_password_length
+
+  private
+  def validate_password_length
```

Comment:
Please run rubocop before pushing PR.

You can also check solution from your [Gemfile](https://github.com/MStodolkiewicz/Monte_Cinema/blob/main/Gemfile#L36) where you have a link to [RubyGuides](https://guides.rubyonrails.org/active_model_basics.html#securepassword)

It look good to me, approved 🎹

Reply:
Thanks. Totally forgot about rubocop on this PR.